### ISSUE TITLE
Bug: Stuck on installing

### ISSUE THREAD

**Issue by gsabater** [ryu1kn/vscode-extension-update-reporter#1]
Hello,

I have used this extension but i have a few issues with it.
First, when i install the extension, the extension is stuck in installing.
After that, no uninstall button is present, even when the user config file has been updated by the extension.

When i try to uninstall, the extension does nothing, but if I restart vscode, the extension is gone.

Thanks

**Comment by ryu1kn** [comment]
Hi @gsabater , can you reproduce the installation issue? I uninstalled/installed the extension on my mac and linux multiple times but never had the issue. If you can, can you let me know the steps to reproduce and your environment (vscode version, your os and its version) so that I can reproduce it on my machine and take a closer look at it?

But, at the same time, I'm not sure if the extension could affect the installation process itself in the first place...

**Comment by gsabater** [comment]
Hello @ryu1kn 

vscode: 1.23.0
os: windows 10

The installation method I tried yesterday is via the vscode plugin interface.
Today I have tried to install via CLI and I put here the error log:

![errror](https://user-images.githubusercontent.com/4948172/39860774-d9018866-543e-11e8-85a3-52db1b89c5eb.JPG)

Hope it helps

**Comment by ryu1kn** [comment]
@gsabater Oh wow, thanks for the quick reply!

I haven't tested the installation from the command line. Just quickly did it on my linux and it worked fine, it seems I need to try it on Windows. Thanks for the information and the screenshot is super helpful!! :+1:

**Comment by ryu1kn** [comment]
@gsabater It seems like installation fails on Windows regardless of whether you install it from command line or UI. On linux, installing from command line worked just fine. Thanks for reporting this!

**Comment by gsabater** [comment]
@ryu1kn No problem, just let me know if I can help in any other way. 
Close the issue when you have the solution so we can know try to use your extension again.

**Comment by ryu1kn** [comment]
Sure thing. I'll keep you updated!

**Comment by ryu1kn** [comment]
Got the same error on downloading the extension (`.vsix` file) and running the following command:

```
code --install-extension .\Downloads\ryu1kn.extension-update-reporter-0.1.1.vsix
```

Also from the log, I can see unnecessary files are not property ignored. `vsce ls` tells me that they're packaging those unnecessary ones.

**Comment by ryu1kn** [comment]
Released as v0.1.2!

```
> code --install-extension ryu1kn.extension-update-reporter
Found 'ryu1kn.extension-update-reporter' in the marketplace.
Installing...
Extension 'ryu1kn.extension-update-reporter' v0.1.2 was successfully installed!
```

**Comment by gsabater** [comment]
Hello @ryu1kn 
This is happening again after 1.0 release

Salut

**Comment by ryu1kn** [comment]
Oh no... that's too bad.
Thank you for reporting the problem. Really appreciated it!

**Comment by ryu1kn** [comment]
Ok, once again it's fixed in v1.0.1. Sorry for the inconvenience caused!
